### ISSUE TITLE
[Question]: Is the KRibbon RTL compliant

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#1569]
Following on from #343, has anyone tested the KRibbon to make sure that it is RTL compliant?

**Comment by giduac** [comment]
Hi @Wagnerp & @Smurf-IV 

Are all (whole TK) controls RTL compliant are is there more work todo to reach this goal?

If there's still a lot of work to do on this than it might be better to turn it into a project and do good inventory and plan before starting off...

**Comment by Wagnerp** [comment]
> Hi @Wagnerp & @Smurf-IV
> 
> Are all (whole TK) controls RTL compliant or is there more work todo to reach this goal?
> 
> If there's still a lot of work to do on this than it might be better to turn it into a project and do good inventory and plan before starting off...

Hi @giduac 

This is what I'm thinking. It would be good to get someone who uses RTL to look into this.

**Comment by Wagnerp** [comment]
Current situation using `nightly` >= .176

https://github.com/Krypton-Suite/Standard-Toolkit/assets/949607/1843ffd5-1350-44e6-9452-a5efeb88bd79

**Comment by giduac** [comment]
@Wagnerp 

Is the ribbon the only control that needs looking into RTL or are there others?

Like said earlier it can be a good way to go about it and take it on as whole to make RTL a complete feature of KT  instead of reacting to tickets. So a bit more of a proactive instead of a  reactive approach.

**Comment by Wagnerp** [comment]
> @Wagnerp
> 
> Is the ribbon the only control that needs looking into RTL or are there others?
> 
> Like said earlier it can be a good way to go about it and take it on as whole to make RTL a complete feature of KT instead of reacting to tickets. So a bit more of a proactive instead of a reactive approach.

@giduac 

Possibly the docking, workspace etc as well.

**Comment by Wagnerp** [comment]
Can confirm that navigator controls work as expected

**Comment by giduac** [comment]
@Wagnerp 
And these isses:
https://github.com/Krypton-Suite/Standard-Toolkit/issues?q=is%3Aissue+is%3Aopen++rtl

**Comment by Wagnerp** [comment]
Superseded by #1570
